example/.github/workflows/orca.yml

```diff
@@ -93,6 +93,8 @@ jobs:
       # @see https://github.com/acquia/orca/blob/main/docs/advanced-usage.md#ORCA_AMPLITUDE_API_KEY
       # ORCA_TELEMETRY_ENABLE: "TRUE"
       #
+      # Enable Nightwatch.js testing. Acceptable values are "TRUE" and "FALSE".
```

Comment:
This comment should reflect that fact that if you set this value to a literal boolean here `orca-enable-nightwatch` won't have any effect anymore on the individual jobs.